packages/calcite-components/src/components/handle/handle.tsx

```diff
@@ -106,6 +106,13 @@ export class Handle implements LoadableComponent, T9nComponent, InteractiveCompo
    */
   @Prop() label: string;
 
+  /**
+   * When `true`, disables unselecting the component when blurred.
+   *
+   * @internal
+   */
+  @Prop() blurUnselectDisabled = false;
```

Comment:
Is there a way to tie this name to `activated`, or would that not be accurate? Just a nitpick as its an internal prop, but its a bit confusing if unfamiliar with the component.

Reply:
I was thinking that but I think the next step is going to be renaming `active` to `selected` and making it public.